examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -6155,6 +6156,7 @@ endpoint 1 {
     emits event Notify;
     ram      attribute mask default = 1;
     ram      attribute state default = 0;
+    ram      attribute supported default = 1;
```

Comment:
default value 0?

Reply:
Default value of Mask should be 1 in case of refrigerator alarm, so if we set Supported to 0 we have to set Mask to 0 too. Also check https://github.com/project-chip/connectedhomeip/pull/27475#issuecomment-1608426871